alerts/cluster-network-operator/SouthboundStaleAlert

```diff
@@ -0,0 +1,23 @@
+# Southbound Stale Alert 
+
+## Meaning 
+
+The alert `SouthboundStale` is triggered when ovn-northd has gone longer than 2 minutes 
+without succesfully synching any changes to the Southbound Database.
+
+## Impact 
+
+Networking control plane is degraded and Networking Configuration updates applied to the cluster will not be applied.
+Existing workloads should continue to have connectivity, but the OVN-Kubernetes control plane and/or the OVN northbound database
+may not be functional. 
+
+## Diagnosis
+
+Either ovn is overloaded, i.e. the load on the cluster is too large due to a large number of services, network policies, pods, or nodes, or the control plane
```

Comment:
This isnt steps to diagnosis this issue.
You need to lay out possibly scenarios and their associated oc commands or whatever to then test to see if the scenario is what is happening.

A few scenarios:
1. Check if SB DB is functional. i.e. its running without errors, and it has a RAFT cluster leader. Connect to it with command ovn-sbctl show or whatever to make sure its ok. I think this command can only be run against the RAFT leader for SB DB. Please confirm this.
2.  OVN Northd isnt functional. Check to see if OVN Northd container is running without errors. Document commands for this.
3. OVN Northd is overloaded. You need to check the CPU usage of northd over a period of time to prove this. Make sure you give them a metric to look at to prove this. If the CPU usage is constantly close to 100% then, its overloaded.
4. OVN Northbound DB is not functioning. Maybe OVN Northd isnt able to consume updates from this DB and thats why it cannot update the SB DB. 

5. Check northds connectivity between OVN NB and SB leaders.

Make sure you list all the oc commands etc for all of this. Assume whomever youre writing this for does not know.